vncviewer/ServerDialog.cxx

```diff
@@ -213,6 +213,7 @@ void ServerDialog::handleSaveAs(Fl_Widget *widget, void *data)
     fl_alert("%s", e.str());
   }
   
+  delete [] servername;
```

Comment:
Can't this also be solved by just getting rid of the strdup()? And if not, `free()` needs to used, not `delete []`.